hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/security/x509/certificate/authority/CertificateServer.java

```diff
@@ -82,7 +82,7 @@ X509Certificate getCertificate(String certSerialId)
    */
   Future<X509CertificateHolder> requestCertificate(
       PKCS10CertificationRequest csr,
-      CertificateApprover.ApprovalType type)
+      CertificateApprover.ApprovalType type, boolean scmCert)
```

Comment:
can we use a enum for role instead with om/scm/dn for now, that might help query certificate by role as well.

Reply:
Yes for now in this PR will only address for SCM, I can open a new Jira to do for other roles. So, that it will help during list with role type.